drivers/wifi/nrf700x/Kconfig

```diff
@@ -574,6 +574,9 @@ config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
 
+menu "Operation Band"
```

Comment:
```suggestion
menu "nRF70 operational band(s)"
```

Reply:
This is not nRF70 specific, maybe "nRF Wi-Fi operation band(s)"